### PR TITLE
Aftershock: Port Augustmoon Start Scenario

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: Create a start scenario within Port Augustmoon"

#### Purpose of change

This adds what will eventually become the default Aftershock exoplanet gameplay scenario by letting players start within the mod's NPC hub 

 You spawn within  the docking arm of port augustmoon like if you had just taken a shuttle there, after that you are set to trade or inmediatelly leave for the planet (either the exoplanet or the vanilla earth). 

#### Describe the solution
Required adding and rewriting some previous professions so the new scenario wasnt completely empty. The same applied to hobbies/backgrounds so I added a few new ones.

#### Testing
Spawned a handfull of characters.

### CODE REVIEW

File: data/mods/Aftershock/player/hobbies.json [hunk 1]
```diff
@@ -8,5 +8,41 @@
     "points": 3,
     "CBMs": [ "afs_bio_chemical_enhancement_rig", "bio_adrenaline", "bio_metabolics", "bio_power_storage_mkII" ],
     "skills": [ { "level": 1, "name": "dodge" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "afs_earthling",
+    "name": "Earth born",
+    "description": "Your mannerisms and accent are a dead giveaway that you where raised on Earth.  Whatever your thoughts about the homeworld might be, the simple fact that you can call it home is likely to affect relationships with people liable to care about such things.",
```
Comment (Maleclypse) [inline data/mods/Aftershock/player/hobbies.json:17]:
```suggestion
    "description": "Your mannerisms and accent are a dead giveaway that you were raised on Earth.  Whatever your thoughts about the homeworld might be, the simple fact that you can call it home is likely to affect relationships with people liable to care about such things.",
```

**Review by Maleclypse — COMMENTED** [review]
Love it!

File: data/mods/Aftershock/player/hobbies.json [hunk 2]
```diff
@@ -8,5 +8,41 @@
...
+    "proficiencies": [ "prof_helicopter_pilot" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "afs_leadership",
+    "name": "Empathic Leadership",
+    "description": "You have a natural talent for leadership and can easily see yourself in others.  Quite literally.  In fact, you can swap control to any other character that follows you.",
+    "points": 3,
+    "traits": [ "AFS_LEADERSHIP" ]
```
[the remark this answers is not in the document]
Comment (John-Candlebury) [reply data/mods/Aftershock/player/hobbies.json:46]: Its so significant gameplay wise that I wanted to shove it into a background skill. Plus its more of a thing you learn than an innate  trait.

File: data/mods/Aftershock/items/tools.json [hunk 3]
```diff
@@ -659,7 +659,53 @@
     "melee_damage": { "bash": 9 }
   },
   {
-    "id": "military_mess_kit",
+    "id": "multi_cooker",
```
[the remark this answers is not in the document]
Comment (John-Candlebury) [reply data/mods/Aftershock/items/tools.json:662]: Yeah its an override.

[the remark this answers is not in the document]
Comment (Fris0uman) [reply data/mods/Aftershock/items/tools.json:662]: It doesn't need a `"copy-from": "multi_cooker"` line to be an override?

[the remark this answers is not in the document]
Comment (John-Candlebury) [reply data/mods/Aftershock/items/tools.json:662]: Oh that's no necessary the same ID is enough.

File: data/mods/Aftershock/items/tools.json [hunk 4]
```diff
@@ -658,6 +658,52 @@
     "flags": [ "ALLOWS_REMOTE_USE" ],
     "melee_damage": { "bash": 9 }
   },
+  {
+    "id": "multi_cooker",
```
Comment (John-Candlebury) [inline data/mods/Aftershock/items/tools.json:662]:
```suggestion
    "id": "multi_cooker",
    "copy-from": "multi_cooker"
```

**Pull request merged**